<launch>

  <!--连接VLP16-->
  <include file="$(find velodyne_pointcloud)/launch/VLP16_points.launch" />

  <!--自车Marker-->
  <node pkg="lidar_grid" type="ego_car_node" name="ego_car" />
  
  <!--激光雷达零方位角标定-->
  <node pkg="lidar_grid" type="azimuth_calibration" name="azimuth_calibration" >
    <rosparam command="load" file="$(find lidar_grid)/param/azimuth_cali.yaml" />
  </node>

  <!--rviz可视化-->
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find lidar_grid)/rviz/lidar_grid.rviz" required="true"/>    

</launch>